### ISSUE TITLE
Any way to change swagger-stats ui page title?

### ISSUE THREAD

**Issue by firefoxNX** [slanatech/swagger-stats#44]
The ui page title shows as "Swagger Stats UI". Is there any way to change this?

https://github.com/slanatech/swagger-stats/blob/master/lib/swsUtil.js#L262

**Comment by sv2** [comment]
Will introduce option for that ... 
In the meantime, you may consider making your own UI page - just serve markup similar to embedded, with adjustments as you need, on different URL in your app. Or you may even embed swagger-stats UI into one of existing UI pages in your app - just add links to css and js, and init swagger-stats UI on specific div:
```
<script>\
$(document).ready(function(){$("#SWSUI").swaggerstatsui({});});\
</script>\
```

**Comment by sv2** [comment]
Starting from 0.99.1 page title shows what is passed in the  "name" configuration option of swagger-stats

**Comment by sv2** [comment]
Addressed v0.99.1
